ntp.toml

```diff
@@ -5,7 +5,7 @@ log-filter = "info"
 log-format = "full"
 
 # Peers can be configured as a simple list (pool servers from ntppool.org)
-peers = ["0.pool.ntp.org", "1.pool.ntp.org", "2.pool.ntp.org", "3.pool.ntp.org"]
+peers = ["0.pool.ntp.org"]
```

Comment:
is this change on purpose?

Reply:
Accident, but while I'm at it, I've changed this to be a pool now instead of the individual peers